app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -80,47 +81,46 @@ class AccessApiFragment : Fragment() {
                 is GetAccountResult.NoAccountFound -> {
                     displaySignedOutState()
                 }
+                is GetAccountError -> {
+                    displayDialog(getString(R.string.msal_exception_title), accountResult.exception?.message ?: accountResult.errorMessage)
+                }
             }
         }
     }
 
     private fun signIn() {
         CoroutineScope(Dispatchers.Main).launch {
+            val email = binding.emailText.text.toString()
+            val password = CharArray(binding.passwordText.length())
+            binding.passwordText.text?.getChars(0, binding.passwordText.length(), password, 0)
+
+            val actionResult: SignInResult
             try {
```

Comment:
Why do we still need this try? Also, a try without a catch?

Reply:
Applies to multiple places.

Reply:
I think it's because there is a try-finally block which has relation with the password security/privacy fix completed by Burak before.

Reply:
I get the point of the try-finally now, but I don't think we need a try at all now anymore. The SDK shouldn't throw an exception, so developers don't need to expect it.

Reply:
Removed try finally block